Add-ons/The-Licensing-model/index.md

```diff
@@ -55,10 +55,14 @@ That this is an add-on domain for existing licenses. Refunds will not be given f
 
 ### Umbraco Deploy
 
-- A single license covers the installation and use of Umbraco Deploy in 1 production domain, as well as 2 development domains
-- The production domain includes all subdomains (e.g. `*.domain.tld`), as well as the `.local` extension (e.g. `domain.tld.local`)
-- The development domains works with or without the `www` subdomain
-- The license also includes `localhost` as a valid domain
+Deploy license:
+
+A single licence covers one Umbraco solution, including all domains hosted by the solution,
+all production environments (if load-balancing) and all non-production environments.
+
+- If you have a solution covering multiple domains- e.g. www.mysite.com and www.mysite.dk - load balanced in production over multiple servers running from the same database, managed from the same back-office instance, and with any number of non-production environments (staging, qa, etc.) - you only need one licence.
+
+- If you have a web presence that consists of two separate websites hosted on different domains or sub-domains - e.g. www.mysite.com and shop.mysite.com - with each of these managed as a separate Umbraco installation using their own database and back-office in production - you would need two licences.
```

Comment:
Instead of a list, where the information needed is at the end of very long sentences, would it make more sense to start with "You only need one license when..." and "You will need more than one license when..."?
What do you think?